SyncUps-StackBased/SyncUps/RecordMeeting.swift

```diff
@@ -22,10 +23,12 @@ class RecordMeetingModel: Hashable {
   @ObservationIgnored
   @Dependency(\.speechClient) var speechClient
 
-  var onMeetingFinished: (String) async -> Void = unimplemented(
-    "RecordMeetingModel.onMeetingFinished"
-  )
+  @DependencyEndpoint
+  @ObservationIgnored 
+  var onMeetingFinished: (_ transcript: String) async -> Void
```

Comment:
Using `@DependencyEndpoint` directly on the model is convenient, but quite verbose with observation, and the order of these macros also matters, which is also not great.

Not sure if there's a better way, though, and still an improvement on `unimplemented`.